pkg/fixtures/fixtures.go

```diff
@@ -297,19 +298,10 @@ func (fxt *Fixture) UpdateEnv() error {
 }
 
 func (fxt *Fixture) makeRequest(ctx context.Context, data FixtureRequest, apiVersion string) ([]byte, error) {
-	var rp requests.RequestParameters
-
-	if data.Method == "post" && !fxt.FixtureData.Meta.ExcludeMetadata {
-		now := time.Now().String()
-		metadata := fmt.Sprintf("metadata[_created_by_fixture]=%s", now)
-		rp.AppendData([]string{metadata})
-	}
-
```

Comment:
My first pass at the idempotency keys was to add `rp.SetIdempotency(idempotencyKey)` here like metadata, but found that tests failed. In fact, these request parameters are entirely unused during the fixture execution path!

Reply:
Thank you so much for fixing this!